views/partials/nhl_player_card

```diff
@@ -0,0 +1,41 @@
+
+<script id="card" type="text/x-handlebars-template">
```

Comment:
what's the point of making this card if it's just a copy/paste of the nba card? We should just add if clauses in the player_card when needed or do another subtemplate where things really start diverging